workflows/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Make sure to order by alphabetical list
+add_holohub_application(real_time_ai_surgical_video_processing DEPENDS OPERATORS aja_source)
```

Comment:
Are we using AJA?

Reply:
Only if the user wants to use AJA as source, right now we have three options: Replayer, AJA, and HSB.

Reply:
Sounds good.